### DISCUSSION TITLE
Issue with Generating /docs/api.json Using dedoc/scramble with Laravel 10

### DISCUSSION THREAD

**Discussion by EVOSandru6** [dedoc/scramble#316] [Q&A]
Hello,

I'm encountering an issue when attempting to generate the api.json file for documentation purposes using dedoc/scramble version ^0.8.5 alongside laravel/framework version ^10.0. When navigating to /docs/api.json, I encounter the error previously described. This issue arises in a setup that's supposed to be supported, given the compatibility of the versions mentioned.

Environment:

Laravel Version: ^10.0
dedoc/scramble Version: ^0.8.5
PHP Version: (include your PHP version, if possible)
Operating System: (include your OS if relevant)
Steps to Reproduce:

Install laravel/framework version ^10.0
Install dedoc/scramble version ^0.8.5
Navigate to /docs/api.json
Expected Behavior:
The expected behavior is for the /docs/api.json to be generated successfully without encountering any errors, providing the API documentation as defined.

Actual Behavior:
Upon navigating to /docs/api.json, I'm met with the described error, preventing the generation of the API documentation.

Additional Information:
(Include any additional information that might help in diagnosing the issue, such as specific error messages, logs, or screenshots.)

I've ensured that all packages are up to date and am following the configuration instructions provided. Has anyone encountered a similar issue, or does anyone have insights on how to resolve this?

Thank you in advance for your assistance.

**Comment by romalytvynenko** [comment]
@EVOSandru6 can you create a Laravel project repository with reproduction of the issue?
